cmd/lncli/cmd_pay.go

```diff
@@ -403,6 +403,53 @@ func sendPaymentRequest(ctx *cli.Context,
 	}
 }
 
+var trackPaymentCommand = cli.Command{
+	Name:      "trackpayment",
+	Category:  "Payments",
+	ArgsUsage: "hash",
+	Action:    actionDecorator(trackPayment),
+}
+
+func trackPayment(ctx *cli.Context) error {
+	args := ctx.Args()
+
+	conn := getClientConn(ctx, false)
+	defer conn.Close()
+
+	client := routerrpc.NewRouterClient(conn)
+
+	if !args.Present() {
+		return fmt.Errorf("hash argument missing")
+	}
+
+	hash, err := hex.DecodeString(args.First())
```

Comment:
Do we want to see if args are provided?

Reply:
At is happening right above, `Args.Present()`

Reply:
Indeed, missed that, was looking for the count :)